rubyau/constitution.md

```diff
@@ -1,468 +1,779 @@
+
+2. The Chairperson of the meeting must declare all positions on the Committee vacant and hold elections for those positions in accordance with rules 51 to 54.
+
+#### 51 Nominations
+
+1. Prior to the election of each position, the Chairperson of the meeting must call for nominations to fill that position. Nominations may be voiced by completing the nomination form and returning it before the meeting.
+
+2. An eligible member of the Association may—
+  1. nominate himself or herself; or
+  2. with the member's consent, be nominated by another member.
+
+3. A member who is nominated for a position and fails to be elected to that position may be nominated for any other position for which an election is yet to be held.
+
+4. Nominations are held twice a year at the AGM and an SGM for each position that expires at that time.
+
+#### 52 Election of President etc.
+
+1. At the annual general meeting and SGM where nominations occur separate elections must be held for each of the following positions—
+  1. President;
+  2. Vice-President;
+  3. Secretary;
+  4. Treasurer.
+
+2. If only one member is nominated for the position, the Chairperson of the meeting must declare the member elected to the position.
+
+3. If more than one member is nominated, a ballot must be held.
+
+#### 53 Election of ordinary members
+
+1. The annual general meeting may by resolution decide the number of ordinary members of the Committee (if any) it wishes to hold office for the next year.
 
-2. an entry to that effect in the minute book of the Association— 
+2. A single election may be held to fill all of those positions in so far as the position is vacant at that meeting.
 
-is evidence of the fact, without proof of the number or proportion of the votes recorded in favour of, or against, that resolution.
+3. If the number of members nominated for the position of ordinary committee member is less than or equal to the number to be elected, the Chairperson of the meeting must declare each of those members to be elected to the position.
 
-## 20. Proxies
+4. If the number of members nominated exceeds the number to be elected, a ballot must be held.
 
-1. Each member is entitled to appoint another member as a proxy by notice given to the Secretary no later than 24 hours before the time of the meeting in respect of which the proxy is appointed. 
+#### 54 Term of office
 
-2. The notice appointing the proxy must be—
-  1. provided to the Secretary via email; or 
-  2. for a meeting of the Association convened under ‘Discipline, suspension and  expulsion of members’ (7), in the form set out in Appendix 2; or 
-  3. in any other case, in the form set out in Appendix 3. 
+1. Subject to subrule 3 and rule 56, a committee member holds office until the positions of the Committee are declared vacant at the next annual general meeting or special general meeting whichever falls on the 12 month anniversary of the member’s appointment.
```

Comment:
Rule 56 is referenced here, but it appears the relevant rule is Rule 55 (Vacation of office). Can someone confirm this is correct and fix this reference?